Packs/SentinelOne/ReleaseNotes/3_2_0.md

```diff
@@ -0,0 +1,16 @@
+#### Integrations
+
+##### SentinelOne v2
+
+- sentinelone-get-alerts - changed created_until mandatory field to optional
+- sentinelone-get-alerts - updated created_from and created_until command arguments to accept date inputs like "10 days", "2 hours","5 months"
+- sentinelone-get-threats - updated created_after, created_before, created_from and created_until command arguments to accept date inputs like "10 days", "2 hours","5 months" 
+- fetch - added the option to fetch threats, alerts or both including additional filtering option
+- sentinelone-get-status - added - returns status of a Deep Visibility Query
+- Updated the Docker image to: *demisto/python3:3.10.11.56082*.
```

Comment:
regarding the first note, created_until in the yml file is defined as required even with your changes
```suggestion
- Updated the *created_until* argument to be optional instead of required in the ***sentinelone-get-alerts*** command.
- Updated the *create_until* and *created_from* arguments to accept input in the form of "10 days", "2 hours","5 months" in the ***sentinelone-get-alerts*** command.
- Updated the *create_after* *created_before* *created_from* and *created_until* arguments to accept input in the form of "10 days", "2 hours","5 months" in the ***sentinelone-get-threats*** command.
- Updated the. ***fetch-incidents*** command to fetch threats, alerts or both, including additional filtering options.
- Added a new command ***sentinelone-get-status***: returns status of a Deep Visibility Query.
- Updated the Docker image to: *demisto/python3:3.10.11.56082*.
```